src/plugins/workspace/public/components/add_collaborators_modal/add_collaborators_modal.tsx

```diff
@@ -84,6 +84,29 @@ export const AddCollaboratorsModal = ({
   const [isAdding, setIsAdding] = useState(false);
 
   const handleAddCollaborators = async () => {
+    const singleStarUserIds =
+      permissionType === 'user'
```

Comment:
So we will allow * for group?

Reply:
It should not be allowed. Thanks for the reminder!